fizzBuzzRedux.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Fizz Buzz Redux</title>
+</head>
+<body>
+  <form id="inputForm" action="javascript:onButtonClick()">
+  	<input type="text" name="input1" id="input1" placeholder="First Word">
```

Comment:
Your `startNumber` and `endNumber` inputs are wonderfully descriptive, yet you use `input1` and `input2` as the names of the inputs for the text strings -- is there a more descriptive name that could be used for these?